mesh/mesh.cpp

```diff
@@ -6349,6 +6355,7 @@ STable3D *Mesh::GetElementToFaceTable(int ret_ftbl)
    return NULL;
 }
 
+/*
 // shift cyclically 3 integers so that the smallest is first
```

Comment:
mention here that this function has been removed as of https://github.com/mfem/mfem/pull/1046

Reply:
This is a static function in `.cpp` file, so it was never available outside this `.cpp` file.

However, it is still needed while we keep `Mesh::ReorientTetMesh()` deprecated with its current implementation.

Reply:
oh above I meant  `Mesh::ReorientTetMesh()` (commented out as part of this comment) not `Rotate3`